### PR TITLE
Add new release labels

### PR DESCRIPTION
**Pull request by richlander**
Apply new labels to `release-index.json` and `release.json`.

This PR will be merged on July 13th.

Context: https://github.com/dotnet/designs/pull/265

We are also changing the `support-phase` property, which is a breaking change at GA time. The property will follow the progression below:

`preview` -> `rc` -> `active` -> `maintenance` -> `eol`

The new `release-type` property will be set as either `lts` or `standard` and will not change, even after EOL.

The spec suggests a string array of labels. On further discussion, that doesn't seem necessary. This simpler approach seems better and more efficient for readers. The only label we're missing is `latest`. It doesn't seem warranted to add another property to account for that.

I updated all the releases in `release-index.json` to make it internally consistent, but then only updated the in-support releases for `release.json`. That's definitely up for discussion. Seemed pragmatic.

@mairaw

### CODE REVIEW

**Comment by rbhanda** [comment]
@mairaw can we test this for dot.net to ensure this doesn’t break it?

**Comment by richlander** [comment]
We know that this change will break the website (mostly due to the same issue as https://github.com/dotnet/core/pull/7497). @mairaw would like to test this change, as you suggest, to see if there are other issues.

If we have other known contacts who use these files, now would be a good time to contact them.

**Comment by richlander** [comment]
> With that change merged, we could even add the .NET 7 EOL dates back in.

This PR does that. We can wait to merge this PR or unrevert the other one. Might as well wait and then we can tell people about the changes all at once.

**Comment by richlander** [comment]
I'll leave that up to you. Whichever approach you prefer.

**Comment by joeloff** [comment]
Are you planning on adding the release-type to older releases.json files, e.g. 1.0 or 2.2? If not, what should be the default value when reading these files for the property? We have use cases for processing these files and I want to make sure the releases API handles this in a consistent fashion. I could also consider an undefined enum.

**Comment by richlander** [comment]
I updated all the release objects in `release-index.json`. I didn't update the old `release.json` files in absence of a reason to do so. If you have a scenario where that would be helpful, I'm willing to do it. It won't take me very long. Should I?

**Comment by joeloff** [comment]
> I updated all the release objects in `release-index.json`. I didn't update the old `release.json` files in absence of a reason to do so. If you have a scenario where that would be helpful, I'm willing to do it. It won't take me very long. Should I?

That would be great, unless you have a suggestion for assigning a default value when the `release-type` is absent in the JSON - I'd be open to that as well.

**Comment by richlander** [comment]
Happy to deal with the merge conflicts. However, I think I promised that I wouldn't merge these files until 7/13 (or some similar date). I'd rather wait until July patch Tuesday. Fair?

**Comment by mairaw** [comment]
Of course, no problem!

**Comment by mairaw** [comment]
I just dealt with the merge conflicts since I also had to add the missing property to all other versions. And when would we add the changes to the support-phase property? I just need to account for the interim solution with the current values.

**Comment by richlander** [comment]
Thank @mairaw!

File: release-notes/releases-index.json [hunk 1]
```diff
@@ -44,6 +47,7 @@
             "latest-runtime": "3.1.26",
             "latest-sdk": "3.1.420",
             "product": ".NET Core",
+            "release-type" : "lts",
```
Comment (joeloff) [inline release-notes/releases-index.json:50]: Just noticed this. The index has 3.1 as LTS and LTS, but the 3.1 releases.json has LTS and maintenance

File: release-notes/releases-index.json [hunk 2]
```diff
@@ -44,6 +47,7 @@
...
             "support-phase": "lts",
```
Comment (richlander) [inline release-notes/releases-index.json:51]:
```suggestion
            "support-phase": "maintenance",
```

File: release-notes/releases-index.json (hunk 1 above)
Comment (richlander) [reply release-notes/releases-index.json:50]: Thanks. Resolved.

**Comment by timheuer** [comment]
Do we have tests or any validation for install scripts?  There are args that expect Channel that will now be changing (https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#options)

**Comment by timheuer** [comment]
Yes @richlander I'm thinking to make sure things like `-Channel sts` would be expected to work...I _think_ it will work based on my 2m reading of the code, but might need to just expand test validation/coverage of adding the new labels.

**Comment by richlander** [comment]
Ah. I see now. I'm not sure there is a big scenario for that. I think the following scenarios make sense:

- Latest release
- Latest LTS
- this x.y release

For example, what would `sts` return now? Seems like an anti-pattern. The logical labels should only exist if they are guaranteed to always return an in-support release. Fair?

**Comment by timheuer** [comment]
I think today `sts` would error because it would try to construct an aka link that doesn't exist. https://github.com/dotnet/install-scripts/blob/main/src/dotnet-install.ps1#L907  Maybe a non-issue, but would be good to make sure things work as expected.  There is 'Current' in places in the code and tests...probably need expansion.

**Comment by richlander** [comment]
I believe that @MattGal and @mmitche were involved in setting up those feeds. Is "Current" the same as latest? 

If so, we may want to undocument that and (compatibly) rename it to "Latest".

**Comment by timheuer** [comment]
Tagging @bekir-ozturk also directly.

**Comment by bekir-ozturk** [comment]
As I understand it, the label `Current` works as expected with install scripts (installs .NET 6.0) while it undesirably refers to 5.0 in VS.

@richlander already mentioned that this will be a breaking change, but I wanted to double check that we are on the same page about what exactly will be broken. So, here are some questions that popped into my head:
1. Does removal of `Current` label also mean the removal of `Current` _channel_? @mmitche @adiaaida 
2. `Current` allows the users of install scripts to say "give me whatever is the latest GA version, whether it is LTS or not" which (warning: speculation ahead!) may be a common scenario for people wanting to stay up to date (end of speculation). If the channel `Current` will indeed be removed, can we say that "this is one of the breaking changes we are introducing"? What alternative can we offer for these users?
3. As we are updating install scripts documentation, do we want to add `sts` to the list of supported channels? Agreeing with @richlander above, it may be an anti-pattern since from the context of the scripts it will mean "give me the latest GA build, but make sure it is one of those that will expire soon".
    I'm unsure whether the benefits of having an `sts` channel in scripts will worth the maintenance costs as well as the potential user confusion.

**Comment by adiaaida** [comment]
> 1. Does removal of `Current` label also mean the removal of `Current` _channel_? @mmitche @adiaaida

`Current` in terms of the concept that users can dotnet-install --version current (ie, the aka.ms links) seem like they would be unaffected by this change. Creating the `current` links is a manually run pipeline that we do once a release (more or less on GA release day). 

>    I'm unsure whether the benefits of having an `sts` channel in scripts will worth the maintenance costs as well as the potential user confusion.

If we add `sts` to dotnet-install we will need to update a bunch of stuff in other places. Please ping me if we decide to do this.

**Comment by richlander** [comment]
Let's just stay as-is. I will write up some docs and probably a blog post on our updated approach. I will review it with you folks to ensure everything is correct.

We _could_ change "Current" to "Latest" but that seems like a lot of churn for little benefit. I am NOT proposing that.

File: release-notes/releases-index.json [hunk 3]
```diff
@@ -8,9 +8,9 @@
             "latest-runtime": "7.0.0-rc.2",
             "latest-sdk": "7.0.100-rc.2.22477.23",
             "product": ".NET",
-            "release-type" : "sts",
-            "support-phase": "preview",
-            "eol-date": "2024-05-14",
```
Comment (richlander) [inline release-notes/releases-index.json]: Why remove the eol-date? Seems good to have.

Comment (mairaw) [reply release-notes/releases-index.json]: We typically add it at RTM when the product is really supported until that date. RC2 is only supported until RTM launches and then you need to switch to RTM.

File: release-notes/releases-index.json [hunk 4]
```diff
@@ -32,6 +34,7 @@
             "latest-runtime": "5.0.17",
             "latest-sdk": "5.0.408",
             "product": ".NET",
+            "release-type" : "standard",
             "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/releases-index.json:38]:
```suggestion
            "support-phase": "eos",
```

File: release-notes/releases-index.json [hunk 5]
```diff
@@ -56,6 +60,7 @@
             "latest-runtime": "3.0.3",
             "latest-sdk": "3.0.103",
             "product": ".NET Core",
+            "release-type" : "standard",
             "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/releases-index.json:64]:
```suggestion
            "support-phase": "eos",
```

File: release-notes/releases-index.json [hunk 6]
```diff
@@ -68,6 +73,7 @@
             "latest-runtime": "2.1.30",
             "latest-sdk": "2.1.818",
             "product": ".NET Core",
+            "release-type" : "lts",
             "support-phase": "eol",
             "eol-date": "2021-08-21",
```
Comment (jamshedd) [inline release-notes/releases-index.json:78]:
```suggestion
            "eos-date": "2021-08-21",
```

File: release-notes/releases-index.json [hunk 7]
```diff
@@ -56,6 +60,7 @@
             "latest-runtime": "3.0.3",
             "latest-sdk": "3.0.103",
             "product": ".NET Core",
+            "release-type" : "standard",
             "support-phase": "eol",
             "eol-date": "2020-03-03",
```
Comment (jamshedd) [inline release-notes/releases-index.json:65]:
```suggestion
            "eos-date": "2020-03-03",
```

File: release-notes/releases-index.json [hunk 8]
```diff
@@ -44,6 +47,7 @@
             "latest-runtime": "3.1.30",
             "latest-sdk": "3.1.424",
             "product": ".NET Core",
+            "release-type" : "lts",
             "support-phase": "maintenance",
             "eol-date": "2022-12-13",
```
Comment (jamshedd) [inline release-notes/releases-index.json:52]:
```suggestion
            "eos-date": "2022-12-13",
```

File: release-notes/releases-index.json (hunk 4 above)
Comment (jamshedd) [inline release-notes/releases-index.json:38]:
```suggestion
            "support-phase": "eos",
```

File: release-notes/releases-index.json [hunk 9]
```diff
@@ -32,6 +34,7 @@
             "latest-runtime": "5.0.17",
             "latest-sdk": "5.0.408",
             "product": ".NET",
+            "release-type" : "standard",
             "support-phase": "eol",
             "eol-date":"2022-05-10",
```
Comment (jamshedd) [inline release-notes/releases-index.json:39]:
```suggestion
            "eos-date":"2022-05-10",
```

File: release-notes/1.0/releases.json [hunk 10]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2019-05-14",
   "latest-runtime": "1.0.16",
   "latest-sdk": "1.1.14",
+  "release-type" : "lts",
   "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/1.0/releases.json:8]:
```suggestion
  "support-phase": "eos",
```

File: release-notes/1.0/releases.json [hunk 11]
```diff
@@ -4,6 +4,7 @@
...
   "eol-date": "2019-06-27",
```
Comment (jamshedd) [inline release-notes/1.0/releases.json:9]:
```suggestion
  "eos-date": "2019-06-27",
```

File: release-notes/1.1/releases.json [hunk 12]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2019-05-14",
   "latest-runtime": "1.1.13",
   "latest-sdk": "1.1.14",
+  "release-type" : "lts",
   "support-phase": "eol",
   "eol-date": "2019-06-27",
```
Comment (jamshedd) [inline release-notes/1.1/releases.json:9]:
```suggestion
  "eos-date": "2019-06-27",
```

File: release-notes/1.1/releases.json [hunk 13]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2019-05-14",
   "latest-runtime": "1.1.13",
   "latest-sdk": "1.1.14",
+  "release-type" : "lts",
   "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/1.1/releases.json:8]:
```suggestion
  "support-phase": "eos",
```

File: release-notes/2.0/releases.json [hunk 14]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2018-07-10",
   "latest-runtime": "2.0.9",
   "latest-sdk": "2.1.202",
+  "release-type" : "standard",
   "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/2.0/releases.json:8]:
```suggestion
  "support-phase": "eos",
```

File: release-notes/2.0/releases.json [hunk 15]
```diff
@@ -4,6 +4,7 @@
...
   "eol-date": "2018-10-01",
```
Comment (jamshedd) [inline release-notes/2.0/releases.json:9]:
```suggestion
  "eos-date": "2018-10-01",
```

File: release-notes/2.1/releases.json [hunk 16]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2021-08-19",
   "latest-runtime": "2.1.30",
   "latest-sdk": "2.1.818",
+  "release-type" : "lts",
   "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/2.1/releases.json:8]:
```suggestion
  "support-phase": "eos",
```

File: release-notes/2.1/releases.json [hunk 17]
```diff
@@ -4,6 +4,7 @@
...
   "eol-date": "2021-08-21",
```
Comment (jamshedd) [inline release-notes/2.1/releases.json:9]:
```suggestion
  "eos-date": "2021-08-21",
```

File: release-notes/2.2/releases.json [hunk 18]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2019-11-19",
   "latest-runtime": "2.2.8",
   "latest-sdk": "2.2.207",
+  "release-type" : "standard",
   "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/2.2/releases.json:8]:
```suggestion
  "support-phase": "eos",
```

File: release-notes/2.2/releases.json [hunk 19]
```diff
@@ -4,6 +4,7 @@
...
   "eol-date": "2019-12-23",
```
Comment (jamshedd) [inline release-notes/2.2/releases.json:9]:
```suggestion
  "eos-date": "2019-12-23",
```

File: release-notes/3.0/releases.json [hunk 20]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2020-02-18",
   "latest-runtime": "3.0.3",
   "latest-sdk": "3.0.103",
+  "release-type" : "standard",
   "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/3.0/releases.json:8]:
```suggestion
  "support-phase": "eos",
```

File: release-notes/3.0/releases.json [hunk 21]
```diff
@@ -4,6 +4,7 @@
...
   "eol-date": "2020-03-03",
```
Comment (jamshedd) [inline release-notes/3.0/releases.json:9]:
```suggestion
  "eos-date": "2020-03-03",
```

File: release-notes/3.1/releases.json [hunk 22]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2022-10-11",
   "latest-runtime": "3.1.30",
   "latest-sdk": "3.1.424",
+  "release-type" : "lts",
   "support-phase": "maintenance",
   "eol-date": "2022-12-13",
```
Comment (jamshedd) [inline release-notes/3.1/releases.json:9]:
```suggestion
  "eos-date": "2022-12-13",
```

File: release-notes/5.0/releases.json [hunk 23]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2022-05-10",
   "latest-runtime": "5.0.17",
   "latest-sdk": "5.0.408",
+  "release-type" : "standard",
   "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/5.0/releases.json:8]:
```suggestion
  "support-phase": "eos",
```

File: release-notes/5.0/releases.json [hunk 24]
```diff
@@ -4,6 +4,7 @@
...
   "eol-date": "2022-05-10",
```
Comment (jamshedd) [inline release-notes/5.0/releases.json:9]:
```suggestion
  "eos-date": "2022-05-10",
```

File: release-notes/6.0/releases.json [hunk 25]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2022-10-11",
   "latest-runtime": "6.0.10",
   "latest-sdk": "6.0.402",
+  "release-type" : "lts",
   "support-phase": "lts",
   "eol-date": "2024-11-12",
```
Comment (jamshedd) [inline release-notes/6.0/releases.json:9]:
```suggestion
  "eos-date": "2024-11-12",
```

File: release-notes/releases-index.json [hunk 26]
```diff
@@ -8,6 +8,7 @@
             "latest-runtime": "7.0.0-rc.2",
             "latest-sdk": "7.0.100-rc.2.22477.23",
             "product": ".NET",
+            "release-type" : "standard",
             "support-phase": "rc",
             "eol-date": null,
```
Comment (jamshedd) [inline release-notes/releases-index.json:13]:
```suggestion
            "eos-date": null,
```

File: release-notes/releases-index.json [hunk 27]
```diff
@@ -20,6 +21,7 @@
             "latest-runtime": "6.0.10",
             "latest-sdk": "6.0.402",
             "product": ".NET",
+            "release-type" : "lts",
             "support-phase": "lts",
             "eol-date": "2024-11-12",
```
Comment (jamshedd) [inline release-notes/releases-index.json:26]:
```suggestion
            "eos-date": "2024-11-12",
```

File: release-notes/releases-index.json (hunk 4 above)
Comment (jamshedd) [inline release-notes/releases-index.json:38]:
```suggestion
            "support-phase": "eos",
```

File: release-notes/releases-index.json (hunk 6 above)
Comment (jamshedd) [inline release-notes/releases-index.json:78]:
```suggestion
            "eos-date": "2021-08-21",
```

File: release-notes/releases-index.json [hunk 28]
```diff
@@ -80,6 +86,7 @@
             "latest-runtime": "2.2.8",
             "latest-sdk": "2.2.207",
             "product": ".NET Core",
+            "release-type" : "standard",
             "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/releases-index.json:90]:
```suggestion
            "support-phase": "eos",
```

File: release-notes/releases-index.json [hunk 29]
```diff
@@ -80,6 +86,7 @@
...
             "eol-date": "2019-12-23",
```
Comment (jamshedd) [inline release-notes/releases-index.json:91]:
```suggestion
            "eos-date": "2019-12-23",
```

File: release-notes/releases-index.json [hunk 30]
```diff
@@ -92,6 +99,7 @@
             "latest-runtime": "2.0.9",
             "latest-sdk": "2.1.202",
             "product": ".NET Core",
+            "release-type" : "standard",
             "support-phase": "eol",
             "eol-date": "2018-10-01",
```
Comment (jamshedd) [inline release-notes/releases-index.json:104]:
```suggestion
            "eos-date": "2018-10-01",
```

File: release-notes/releases-index.json [hunk 31]
```diff
@@ -104,6 +112,7 @@
             "latest-runtime": "1.1.13",
             "latest-sdk": "1.1.14",
             "product": ".NET Core",
+            "release-type" : "lts",
             "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/releases-index.json:116]:
```suggestion
            "support-phase": "eos",
```

File: release-notes/releases-index.json [hunk 32]
```diff
@@ -104,6 +112,7 @@
...
             "eol-date": "2019-06-27",
```
Comment (jamshedd) [inline release-notes/releases-index.json:117]:
```suggestion
            "eos-date": "2019-06-27",
```

File: release-notes/releases-index.json [hunk 33]
```diff
@@ -116,6 +125,7 @@
             "latest-runtime": "1.0.16",
             "latest-sdk": "1.1.14",
             "product": ".NET Core",
+            "release-type" : "lts",
             "support-phase": "eol",
             "eol-date": "2019-06-27",
```
Comment (jamshedd) [inline release-notes/releases-index.json:130]:
```suggestion
            "eos-date": "2019-06-27",
```

File: release-notes/releases-index.json [hunk 34]
```diff
@@ -116,6 +125,7 @@
             "latest-runtime": "1.0.16",
             "latest-sdk": "1.1.14",
             "product": ".NET Core",
+            "release-type" : "lts",
             "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/releases-index.json:129]:
```suggestion
            "support-phase": "eos",
```

File: release-notes/releases-index.json (hunk 32 above)
Comment (jamshedd) [inline release-notes/releases-index.json:117]:
```suggestion
            "eos-date": "2019-06-27",
```

File: release-notes/releases-index.json [hunk 35]
```diff
@@ -92,6 +99,7 @@
             "latest-runtime": "2.0.9",
             "latest-sdk": "2.1.202",
             "product": ".NET Core",
+            "release-type" : "standard",
             "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/releases-index.json:103]:
```suggestion
            "support-phase": "eos",
```

File: release-notes/releases-index.json [hunk 36]
```diff
@@ -68,6 +73,7 @@
             "latest-runtime": "2.1.30",
             "latest-sdk": "2.1.818",
             "product": ".NET Core",
+            "release-type" : "lts",
             "support-phase": "eol",
```
Comment (jamshedd) [inline release-notes/releases-index.json:77]:
```suggestion
            "support-phase": "eos",
```

**Review by jamshedd — CHANGES_REQUESTED** [review]

**Comment by mairaw** [comment]
@jamshedd I resolved all the comments because changing those is no longer the plan of record. Please review again.

**Review by jamshedd — APPROVED** [review]

**Review by leecow — APPROVED** [review]
Good to go - we'll merge on release day.

**Comment by mairaw** [comment]
These changes should be included with all other releases.json changes for GA. We won't be able to handle different changes coming at different PRs for the website (or we'll need to skip some deployments).

File: release-notes/releases-index.json [hunk 37]
```diff
@@ -8,6 +8,7 @@
             "latest-runtime": "7.0.0-rc.2",
             "latest-sdk": "7.0.100-rc.2.22477.23",
             "product": ".NET",
+            "release-type" : "standard",
```
Comment (mairaw) [inline release-notes/releases-index.json:11]:
```suggestion
            "release-type" : "sts",
```

File: release-notes/releases-index.json [hunk 38]
```diff
@@ -32,6 +34,7 @@
             "latest-runtime": "5.0.17",
             "latest-sdk": "5.0.408",
             "product": ".NET",
+            "release-type" : "standard",
```
Comment (mairaw) [inline release-notes/releases-index.json:37]:
```suggestion
            "release-type" : "sts",
```

File: release-notes/releases-index.json [hunk 39]
```diff
@@ -56,6 +60,7 @@
             "latest-runtime": "3.0.3",
             "latest-sdk": "3.0.103",
             "product": ".NET Core",
+            "release-type" : "standard",
```
Comment (mairaw) [inline release-notes/releases-index.json:63]:
```suggestion
            "release-type" : "sts",
```

File: release-notes/releases-index.json [hunk 40]
```diff
@@ -80,6 +86,7 @@
             "latest-runtime": "2.2.8",
             "latest-sdk": "2.2.207",
             "product": ".NET Core",
+            "release-type" : "standard",
```
Comment (mairaw) [inline release-notes/releases-index.json:89]:
```suggestion
            "release-type" : "sts",
```

File: release-notes/releases-index.json [hunk 41]
```diff
@@ -92,6 +99,7 @@
             "latest-runtime": "2.0.9",
             "latest-sdk": "2.1.202",
             "product": ".NET Core",
+            "release-type" : "standard",
```
Comment (mairaw) [inline release-notes/releases-index.json:102]:
```suggestion
            "release-type" : "sts",
```

File: releases-index.json [hunk 42]
```diff
@@ -8,7 +8,7 @@
             "latest-runtime": "7.0.0-rc.2",
             "latest-sdk": "7.0.100-rc.2.22477.23",
             "product": ".NET",
-            "support-phase": "rc",
+            "support-phase": "go-live",
```
Comment (mairaw) [inline releases-index.json:11]: don't introduce the breaking changes now

```suggestion
            "support-phase": "rc",
```

File: release-notes/6.0/releases.json [hunk 43]
```diff
@@ -4,6 +4,7 @@
   "latest-release-date": "2022-10-11",
   "latest-runtime": "6.0.10",
   "latest-sdk": "6.0.402",
+  "release-type" : "lts",
   "support-phase": "lts",
```
Comment (leecow) [inline release-notes/6.0/releases.json:8]: @mairaw - release-type should be 'active'. Is it OK to make that change for the 'catch-up' merge today?

Comment (mairaw) [reply release-notes/6.0/releases.json:8]: yes, I've been testing to what's in the private repo so those are the changes I'm expecting to see and I'm prepared to handle

File: release-notes/releases-index.json [hunk 44]
```diff
@@ -20,6 +21,7 @@
             "latest-runtime": "6.0.10",
             "latest-sdk": "6.0.402",
             "product": ".NET",
+            "release-type" : "lts",
             "support-phase": "lts",
```
Comment (mairaw) [inline release-notes/releases-index.json:25]:
```suggestion
            "support-phase": "active",
```

File: release-notes/releases-index.json [hunk 45]
```diff
@@ -8,6 +8,7 @@
             "latest-runtime": "7.0.0-rc.2",
             "latest-sdk": "7.0.100-rc.2.22477.23",
             "product": ".NET",
+            "release-type" : "sts",
             "support-phase": "rc",
```
Comment (mairaw) [inline release-notes/releases-index.json:12]:
```suggestion
            "support-phase": "go-live",
```

**Pull request merged**